03_Modules/Configuration/src/module/api.ts

```diff
@@ -386,6 +386,42 @@ export class ConfigurationModuleApi
     };
   }
 
+  @AsDataEndpoint(
+      Namespace.LocalAuthList,
+      HttpMethod.Get,
+      ChargingStationKeyQuerySchema,
+      UpdateLocalAuthListSchema,
```

Comment:
You are passing him a schema that is being interpreted as a body. But body's aren't allowed on `GET` probably a mix up with the `PUT` below. Also are you sure about the query schema here? Seems we wouldn't want the objects around idToken to be required? Only stationId.